products/casp/main.pm

```diff
@@ -103,19 +108,27 @@ sub load_feature_tests {
 }
 
 # ==== Installation workflow ====
-load_boot_tests;
+if (get_var('AUTOYAST')) {
+    load_boot_tests;
+    load_autoyast_tests;
+    loadtest 'casp/first_boot';
+    load_feature_tests;
```

Comment:
If you want to run feature tests after autoyast installation set test variable EXTRA=FEATURES